benchmarks/inference-server/vllm/README.md

```diff
@@ -0,0 +1,120 @@
+# Benchmark vLLM on GKE
+
+This stage deploys the [vLLM](https://docs.vllm.ai/en/latest/index.html) inference server.
+
+## Instructions
+
+### Step 1: create and configure terraform.tfvars
+
+Create a `terraform.tfvars` file. `sample-terraform.tfvars` is provided as an
+example file. You can copy the file as a starting point. Note that you will have
+to change the existing `credentials_config`.
+
+```bash
+cp sample-terraform.tfvars terraform.tfvars
+```
+
+Fill out your `terraform.tfvars` with the desired model and server
+configuration, referring to the list of required and optional variables
+[here](#variables). Variables `credentials_config` are required.
```

Comment:
there's no variables header - consider using terraform-docs to create the related table. Or remove this reference for now.

Reply:
Done